charts/seed-monitoring/charts/core/charts/prometheus/values.yaml

```diff
@@ -278,17 +262,7 @@ allowedMetrics:
   - vpa_status_recommendation
   - vpa_spec_container_resource_policy_allowed
   - vpa_metadata_generation
-  hvpa:
```

Comment:
@ggaurav10 I agree that we may not have an HVPA dashboard right away. But HVPA metrics will be essential for us to evaluate the HVPA behaviour especially after it is enabled in canary.

@wyb1 Could you please retain the HVPA metrics? At least until HVPA becomes widely used across landscapes.

Reply:
I have readded the hvpa metrics. Once we have more experience with hvpa we can revisit these metrics and decide if still need/want them.